src/aics-image-viewer/components/App/index.tsx

```diff
@@ -586,7 +594,7 @@ export default class App extends React.Component<AppProps, AppState> {
     // if we want to keep the current control points
     if (thisChannelsSettings.controlPoints && keepLuts) {
       const lut = controlPointsToLut(thisChannelsSettings.controlPoints);
-      aimg.setLut(channelIndex, new Uint8Array(lut.buffer));
+      aimg.setLut(channelIndex, lut);
```

Comment:
introduces a possible bug if an array was being copied before but is not copied anymore?  I'm not SURE this would create a bug but it certainly could violate some assumption downstream if the array data is modified later.

Reply:
nevermind.. I read it too quickly.  I guess it was wrapping the same buffer so behavior shouldn't change if array data is modified. 
But I'm leaving the comment in because its always something to watch out for!